.github/workflows/rc.yml

```diff
@@ -0,0 +1,53 @@
+name: Create Pre-release
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+'
+
+env:
+  CHART_DIR: "charts/common"
+  HELM_REPO_BASE_URL: https://activiti.github.io/activiti-cloud-helm-charts
+  HELM_REPO: Activiti/activiti-cloud-helm-charts
+  HELM_REPO_BRANCH: gh-pages
+  HELM_REPO_SUBFOLDER: .
+
+jobs:
+  prerelease:
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ github.ref_name }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - id: helm-release-and-publish
+        name: Release and publish helm chart
+        uses: Alfresco/alfresco-build-tools/.github/actions/helm-release-and-publish@v1.23.0
+        with:
+          version: ${{ env.VERSION }}
+          chart-dir: ${{ env.CHART_DIR }}
+          helm-repository: ${{ env.HELM_REPO }}
+          helm-repository-branch: ${{ env.HELM_REPO_BRANCH }}
+          helm-repository-subfolder: ${{ env.HELM_REPO_SUBFOLDER }}
+          helm-repository-base-url: ${{ env.HELM_REPO_BASE_URL }}
+          helm-repository-token: ${{ secrets.BOT_GITHUB_TOKEN }}
+          git-username: ${{ secrets.BOT_GITHUB_USERNAME }}
+          do-push: ${{ github.event_name == 'push' }}
```

Comment:
I'm wondering if this condition is needed: the workflow is triggered on `push.tag`, so my understanding is that `gihub.event_name` is always going to be `push`. Am I missing something?